app/src/main/java/org/p2p/wallet/home/ui/select/bottomsheet/NewSelectTokenBottomSheet.kt

```diff
@@ -0,0 +1,64 @@
+package org.p2p.wallet.home.ui.select.bottomsheet
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.FragmentManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import org.koin.android.ext.android.inject
+import org.p2p.uikit.utils.attachAdapter
+import org.p2p.wallet.R
+import org.p2p.wallet.common.analytics.interactor.ScreensAnalyticsInteractor
+import org.p2p.wallet.common.ui.bottomsheet.BaseDoneBottomSheet
+import org.p2p.wallet.common.ui.recycler.adapter.DividerItemDecorator
+import org.p2p.wallet.home.model.Token
+import org.p2p.wallet.home.ui.select.NewSelectTokenAdapter
+import org.p2p.wallet.moonpay.analytics.BuyAnalytics
+import org.p2p.wallet.utils.args
+import org.p2p.wallet.utils.withArgs
+
+private const val EXTRA_ALL_TOKENS = "EXTRA_ALL_TOKENS"
+
+class NewSelectTokenBottomSheet : BaseDoneBottomSheet() {
+
+    companion object {
+        fun show(
+            fm: FragmentManager,
+            title: String,
+            tokens: List<Token>,
+            requestKey: String,
+            resultKey: String
+        ) = NewSelectTokenBottomSheet().withArgs(
+            ARG_TITLE to title,
+            EXTRA_ALL_TOKENS to tokens,
```

Comment:
List<*> is parcelable? O_o

Reply:
List<*> ????

Reply:
we can pass List<Anything> to bundle? i didn't know

Reply:
yes =) list of parcelable